### PR TITLE
3.46.0: Adds new SDK versions and contract files

### PR DESCRIPTION
**Pull request by aavasthy**
# Pull Request Template

## Description

This bumps the SDK GA version from 3.45.2 to 3.46.0 and preview version from 3.46.0-preview.2 to 3.47.0-preview.0. No changes in the public GA contract and preview contract. The images are given below:

image

image
## Type of change

- [X] New feature (non-breaking change which adds functionality)

## Closing issues

To automatically close an issue: closes #IssueNumber

### CODE REVIEW

**Review by ananth7592 — APPROVED** [review]

**Comment by kundadebdatta** [comment]
> No changes in the public GA contract. The preview contract changes are given below:

Curious - Are there any changes in the preview contract ? The screenshot doesn't show any. If there are none, can you please fix the description ?

**Comment by aavasthy** [comment]
> > No changes in the public GA contract. The preview contract changes are given below:
> 
> Curious - Are there any changes in the preview contract ? The screenshot doesn't show any. If there are none, can you please fix the description ?

Updated the comment. No changes in preview as well.

**Review by kundadebdatta — APPROVED** [review]
LGTM!

File: changelog.md [hunk 1]
```diff
@@ -15,6 +15,21 @@ Preview features are treated as a separate branch and will not be included in th
...
+### <a name="3.47.0-preview.0"/> [3.47.0-preview.0](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.47.0-preview.0) - 2024-11-15
+
+### <a name="3.46.0"/> [3.46.0](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.46.0) - 2024-11-15
+
+#### Added
+
+- [4854](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4854) Open Telemetry: Adds open telemetry based versioning.
+
+#### Fixed
+- [4839](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4839) Dependencies: Removes direct reference for Newtonfoft package and add a target build.
```
Comment (kirankumarkolli) [inline changelog.md:27]: Lets please move this to Added section.
Also add a NOTE with breaking aspects to be very explicit (copy snippets from PR description)

Comment (aavasthy) [reply changelog.md]: updated with a note and more description.

**Review by kirankumarkolli — APPROVED** [review]

**Review by ananth7592 — APPROVED** [review]

**Review by kirankumarkolli — APPROVED** [review]

**Review by kundadebdatta — APPROVED** [review]

**Pull request merged**